packages/extension-browser/src/devtools/views/pages/analyze.css

```diff
@@ -33,8 +33,18 @@
 .messages {
     align-items: center;
     display: grid;
-    grid-template-columns: 50% 50%;
-    width: 100%;
+    gap: 1.5rem; /* 24px */
+    margin: 1.5rem 0; /* 24px */
+}
+
+/* Horizontal layout */
+@media (min-width: 36rem /* 576px */) {
```

Comment:
To keep the top of the Nellie graphic from getting too close to the text in the collapsed view, we should move the `.image { margin-top: -10% }` block inside this media query.